edgexfoundry/edgex-examples
# refactor: Upgade cloud-event app service to use V3

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:


## Testing Instructions
run `go test ./pkg/transforms`
start up edgex using no-secty
run `/app-service -cp -dev`
verify service starts
